components/pictureTile/index.tsx

```diff
@@ -0,0 +1,74 @@
+import React, {Fragment, useState} from 'react';
+import {IconButton, ImageListItem, ImageListItemBar, Tooltip} from "@mui/material";
+import FavoriteBorderIcon from "@mui/icons-material/FavoriteBorder";
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import {useFavourite} from "../../hooks/useFavourite";
+import PictureDialog from "../pictureDialog";
+
+interface PictureTileProps {
+  item: ApodType;
+}
+
+export interface ApodType {
+  url: string;
+  title: string;
+  copyright: string;
+  media_type: string;
+  thumbnail_url: string;
+  explanation: string;
+  date: string;
+  hdurl: string;
+}
+
+
+function PictureTile({item}: PictureTileProps) {
+  const {isFavourite, toggleFavourite} = useFavourite(item.date);
+
+  const [open, setOpen] = useState(false);
+
+
+  const handleClickingFavourite = (e: React.MouseEvent<HTMLElement>) => {
+    toggleFavourite(item.date);
+    e.stopPropagation();
+  }
+
+  return (
+      <Fragment><ImageListItem onClick={() => setOpen(true)}>
+        {/* eslint-disable-next-line @next/next/no-img-element */}
+        <img
+            src={item.media_type === "image" ? `${item.url}?w=164&h=164&fit=crop&auto=format` : `${item.thumbnail_url}?w=164&h=164&fit=crop&auto=format`}
```

Comment:
I would move this logic of determining a `src` to separate function, so the markup will be bit cleaner. It's also easier to test this way because you won't need to render the component and check its `src` attribute, you just check the return value of a function